docs/overview.md

```diff
@@ -0,0 +1,202 @@
+priority of that configuration.
+
+Please note that configuration is hierarchical. The last configuration source
+loaded that contains a configuration value will be the value that's used. This
+means that if the same configuration value is found in both `appsettings.json`
+and as an environment variable, the value from the environment variable will be
+the value that's applied.
+
+### Configuration options
+
+See [configuration.md](/docs/configuration.md) for details about how to
+configure specific components of the application.
+
+#### Environment variables
+
+When adding values via environment variables, we recommend that you use the
+double underscore form, rather than the colon form.
+Colons won't work in non-windows environment. For example:
+
+```bash
+  DataSources__EmissionsDataSource="WattTime"
+```
+
+Note that double underscores are used to represent dotted notation or child
+elements that you see in the JSON below. For example, to set proxy information
+using environment variables, you'd do this:
+
+```bash
+  DataSources__Configurations__WattTime__UseProxy
+```
+
+#### Local project settings
+
+For local-only settings you can use environment variables,
+[the Secret Manager tool](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows#secret-manager)
+, or an untracked Development appsettings file to override
+the default project settings.
+
+To use the settings file, rename a copy of the template called
+`appsettings.Development.json.template` to `appsettings.Development.json` and
+remove the first line of (invalid) comments.
+Then update any settings according to your preferences.
+
+> Wherever possible, the projects leverage the
+> [default .NET configuration](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0#default-application-configuration-sources)
+> expectations. Thus, they can be configured using any file matching the
+> format: `appsettings.<ENV>.json`. Where `<ENV>` is the value of
+> the `ASPNETCORE_ENVIRONMENT` environment variable. By convention projects
+> tend to use the provided HostEnvironment constants
+> `Development`, `Staging`, and `Production`.
+
+## Publish WebAPI with container
+
+You can publish Web API for Carbon Aware SDK with container. These instructions
+show how to build / run container image with [Podman](https://podman.io/).
+
+### Build container image
+
+Following commands build the container which named to `carbon-aware-sdk-webapi`
+from sources.
+
+```bash
+$cd src
+$podman build -t carbon-aware-sdk-webapi -f CarbonAware.WebApi/src/Dockerfile .
+```
+
+### Run Web API container
+
+Carbon Aware SDK Web API publishes the service on Port 80, so you need to map
+it to local port. Following commands maps it to Port 8080.
+
+You also need to configure the SDK with environment variables. They are minimum
+set when you use WattTime or ElectricityMaps as a data source.
+
+```bash
+$ podman run -it --rm -p 8080:80 \
```

Comment:
@Kpopovs1 I was looking at the comments in the issue #230, and looks like we are missing the following config in the examples below. Can you add it so that it unblocks the users from using it?
`DataSources__Configurations__WattTime__Type="WattTime"`

Reply:
pushed the fix, thank you for catching this!